sonic-pi-no-libstdc++.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/app/api/CMakeLists.txt b/app/api/CMakeLists.txt
+index bc06d1661..f9cd06885 100644
+--- a/app/api/CMakeLists.txt
++++ b/app/api/CMakeLists.txt
+@@ -99,7 +99,6 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+ find_package(crossguid CONFIG REQUIRED)
+ target_link_libraries(${PROJECT_NAME}
+     PUBLIC
+-        stdc++fs
+         rt
+     PRIVATE
+         crossguid
```

Comment:
This is the thing that had me puzzled for a while because I know nothing about C++ :sweat_smile: 

From what I could understand via some googling, etc., is that with gcc >= 9 (which is a bit old and the flatpak in any case doesn't use), linking this library is not necessary. In general, it is also harmless because it's present in most systems, but it turns out, in the flatpak sdk it is not. I'll also submit that patch to the Sonic Pi project, but it is a bit more involved than here because there gcc 9 needs to be supported I guess so I need to learn how to write that condition in CMake (I've see it's possible).

Reply:
it exist because there are still too many people runing antiquate distro like Ubuntu LTS. Flatpak doesn't ship the .a stub while distro do, so it fails. It's a recurring problem in a world of developer that think Linux == Ubuntu